build

```diff
@@ -25,12 +25,20 @@ if [ "$TARGET" == "vm" ]; then
     --arg buildLive false \
     --arg buildDisk false)
 
-  echo
-  echo "Run ./result/bin/run-in-vm to start a VM."
-  echo
-  echo "- unfocus: ctrl-alt-g"
-  echo "- quit: ctrl-alt-q"
-  echo "- tty: ctrl-alt-2, sendkey ctrl-alt-f1, ctrl-alt-1"
+echo -e "
+Run ./result/bin/run-in-vm to start a VM.
+
+QEMU Shortcuts:
+- Grab/release cursor: ctrl-alt-g
+- Quit: ctrl-alt-q
+
+VM shortcuts:
+Switch to QEMU monitor (ctrl-alt-2), activate shortcut via 'sendkey <some-shortcut>', then switch back to QEMU display (ctrl-alt-1).
+- TTY: ctrl-alt-f1
+- Status console: ctrl-alt-f8
+- Graphical UI: ctrl-alt-f7
+- Toggle controller: ctrl-shift-f12
```

Comment:
These are not so much VM shortcuts, maybe move to a separate section?

The manual also explains these, should we link it more prominently?

Reply:
> These are not so much VM shortcuts, maybe move to a separate section?

I'm not sure I addressed this point in the way you meant, but I renamed the section:
[Rename shortcut section in VM hints](https://github.com/dividat/playos/pull/136/commits/1cb9932824d08f93be7073e69aa084709d192cd9)

Added a link to the manual:
[Link user manual in main readme](https://github.com/dividat/playos/pull/136/commits/df7f02286d7a82f574ec5034655be07fcade3ac8)

Reply:
Toggling the controller works just fine for me with the regular key combination, also in QEMU. Does it not for you?

I was thinking to just move the sendkey explanation to the block above and let the PlayOS shortcut explanations stand on their own.

Reply:
https://github.com/dividat/playos/pull/136/commits/ac243d271b7636891b4d54bddc98a35ff422d33e